### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by NikkiNext**
Patch

## Purpose

Add new translation.

## Proposed changes

**Add translation for:**

* [THEMES]: Lunar, Lunar

## TODO

This is an improvement to support the Russian language in the interface

### CODE REVIEW

File: media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{
+5000, "Lunar"
+5001, "Визуальный Стиль"
+5002, "Команда ReactOS"
+5003, "Cernodile и Illen, основан на базе Lautus"
+5004, "Выпущен под GNU/GPL 2.0, 2019"
+5005, "https://reactos.org/"
+5006, "1.1"
+5007, "Визуальный стиль для ReactOS"
+}
```
Comment (JoachimHenze) [inline media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc:37]: Hi Nikki^^. Please add a newline at the end of the file.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

File: media/themes/Lunar/lunar.msstyles/lunar.rc [hunk 2]
```diff
@@ -196,3 +196,6 @@ NORMAL_NORMALGROUPEXPAND_BMP BITMAP "bitmaps/dummy.bmp"
 #ifdef LANGUAGE_ZH_TW
     #include "lang/zh-TW.rc"
 #endif
+#ifdef LANGUAGE_RU_RU
```
Comment (HBelusca) [inline media/themes/Lunar/lunar.msstyles/lunar.rc:199]: Please move this before ZH_TW to maintain lexicographical ordering.

**Review by HBelusca — COMMENTED** [review]
Cc @reactos/lang-russian

File: media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc [hunk 3]
```diff
@@ -0,0 +1,38 @@
+LANGUAGE LANG_RUSSIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+{
+1000, "ReactOS"
```
Comment (oleg-dubinskiy) [inline media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc:5]: Add 4 extra whitespaces here and in all other lines inside the brackets, to make the strings more readable. Although since it isn't done in other translations, then maybe do it for all languages in a separate PR.

**Review by oleg-dubinskiy — APPROVED** [review]
Looks good now, but I still have a trivial suggestion.

**Review by HBelusca — COMMENTED** [review]
Should be fine now.

Please note, that albeit we don't really enforce this for translations only, for PRs containing actual code changes we require our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
(Note that we aren't the only project asking such requirements, see e.g. Wine. For code changes this is required for legal purposes, due to the nature of the project.)

**Review by JoachimHenze — APPROVED** [review]

**Review by HBelusca — APPROVED** [review]

File: media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+4001, "Крупный шрифт"
+4002, "Огромный шрифт"
+}
+
+STRINGTABLE
+{
+5000, "Lunar"
+5001, "Визуальный Стиль"
+5002, "Команда ReactOS"
+5003, "Cernodile и Illen, основан на базе Lautus"
```
Comment (binarymaster) [inline media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc:32]:
```suggestion
5003, "Cernodile и Illen, основан на Lautus"
```

**Review by binarymaster — COMMENTED** [review]
One minor thing to improve (posted a suggestion); Also, despite we don't require setting up a full legal name for authoring translations, it's highly recommended nevertheless.

File: media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc (hunk 3 above)
Comment (binarymaster) [reply media/themes/Lunar/lunar.msstyles/lang/ru-RU.rc:5]: Not required.

**Review by binarymaster — APPROVED** [review]
